log/logext.go

```diff
@@ -17,16 +19,16 @@ const (
 	// order they appear (the order listed here) or the format they present (as
 	// described in the comments).  A colon appears after these items:
 	//	2009/0123 01:23:23.123123 /a/b/c/d.go:23: message
-	Ldate         = 1 << iota                     // the date: 2009/0123
-	Ltime                                         // the time: 01:23:23
-	Lmicroseconds                                 // microsecond resolution: 01:23:23.123123.  assumes Ltime.
-	Llongfile                                     // full file name and line number: /a/b/c/d.go:23
-	Lshortfile                                    // final file name element and line number: d.go:23. overrides Llongfile
-	Lmodule                                       // module name
```

Comment:
- don't remove deprecated public symbol so that who depended on it can compile passed (but can implement in reasonable way).
- and you can declare it deprecated.